cockroachcloud/export-metrics.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Export Metrics From a CockroachDB dedicated Cluster
+summary: Export Metrics From a CockroachDB dedicated Cluster
+toc: true
+docs_area: manage
+---
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure metrics export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/). Once the export is configured, metrics will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
+
+The metrics export feature to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters created after August 11, 2022. Metrics export to Datadog is supported on all {{ site.data.products.dedicated }} clusters regardless of creation date.
```

Comment:
> The metrics export feature to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters created after August 11, 2022

nit: should we also call out that it's only available on CockroachDB Clusters *running* in AWS? Or is that kind of obvious/assumed?